product-service/serverless.ts

```diff
@@ -0,0 +1,39 @@
+import type { AWS } from '@serverless/typescript';
+
+import getProductsList from '@functions/getProductsList';
+import getProductById from '@functions/getProductById';
+
+const serverlessConfiguration: AWS = {
+  service: 'product-service',
+  frameworkVersion: '3',
+  plugins: ['serverless-esbuild'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs14.x',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+      shouldStartNameWithService: true,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      NODE_OPTIONS: '--enable-source-maps --stack-trace-limit=1000',
+    },
+  },
+  // import the function via paths
+  functions: { getProductsList, getProductById },
+  package: { individually: true },
+  custom: {
+    esbuild: {
+      bundle: true,
+      minify: false,
```

Comment:
One major benefit of using bundlers is minification which helps reduce load times.